package.json

```diff
@@ -35,6 +36,7 @@
     "@mojaloop/central-services-error-handling": "7.3.0",
     "@mojaloop/central-services-metrics": "5.2.0",
     "@mojaloop/central-services-shared": "7.4.3",
+    "axios": "^0.19.0",
```

Comment:
please remove caret from version. also please update all dependencies to latest stable release and update the version in package.json